openedx/features/learner_analytics/views.py

```diff
@@ -93,7 +93,9 @@ def get(self, request, course_id):
                 'weekly_active_users': self.get_weekly_course_activity_count(course_key),
                 'week_streak': self.consecutive_weeks_of_course_activity_for_user(
                     request.user.username, course_key
-                )
+                ),
+                'passing_grade': passing_grade,
+                'percent_grade': percent_grade
```

Comment:
@AlasdairSwan: I think this is a float.  You cool with multiplying by 100 and rounding client-side. :)

Reply:
@AlasdairSwan: FYI: It is now a whole number.

Reply:
Sure, that works @robrap